SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift

```diff
@@ -28,7 +28,7 @@ public final class PushNotificationAPI : NSObject {
     }
 
     // MARK: - Settings
-    public static let server = "https://live.apns.getsession.org"
```

Comment:
Please revert the notification server change

Reply:
Done.